Assets/Mona/QualityAssurance/QualityAssurance.Utils.cs

```diff
@@ -35,7 +35,38 @@ public static GameObject FindParentWithTag(String tagToFind, GameObject starting
             }
             return null;
         }
-    }
-}
 
-#endif
\ No newline at end of file
+        public static void InitDescriptions(){
+            if(ErrorDescriptions != null) return;
+            ErrorDescriptions = new Dictionary<string, string>();
```

Comment:
We might want to call this `ErrorDescriptionMap` so we can use "ErrorDescriptions" as its own struct to separate codes from descriptions